src/tests/ftest/launch.py

```diff
@@ -2656,6 +2656,12 @@ def _process_core_files(self, test_job_results, test):
             self._fail_test(self.result.tests[-1], "Process", message, sys.exc_info())
             return 256
 
+        # One of the dfuse tests intermittently creates core files which is known so make a
+        # special case for that test.
+        if str(test) == './dfuse/daos_build.py' and './conftest' in core_file_processing.exe_names:
+            logger.info('conftest core file exists from daos_build test, ignoring')
+            corefiles_processed = len(core_file_processing.exe_names) - 1
```

Comment:
This did work although the local value of **corefiles_processed** might have been wrong in absolute terms.  If conftest was in the set then it would be the number of executables that created corefiles of interest and the variable was then compared against zero.

I've reworked this PR with your suggestions however.